src/components/BeersList.js

```diff
@@ -0,0 +1,52 @@
+import './BeersList.css';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+
+const BeersList = () => {
+    const baseUrl = "https://ih-beers-api2.herokuapp.com/beers";
+    const [beers, setBeers] = useState([]);
+    const [query, setQuery] = useState("");
+
+    const handleChange = (e) => {
+        setQuery(e.target.value)
+    }
+
+    const renderBeers = () => {
+        return beers.map(beer => {
+            return <div className="Beer" key={beer._id}>
+                <Link to={`/beers/${beer._id}`}>
+                    <div className="img-container">
+                        <img src={beer.image_url} alt={beer.name}/>
+                    </div>
+                    <div className="text-container">
+                        <h2>{beer.name}</h2>
+                        <h3>{beer.tagline}</h3>
+                        <p>Create by: <span>{beer.contributed_by.split(" <")[0]}</span></p>
```

Comment:
That one can break if someone doesn't send the `contributed_by` in the form, one way it would be to add conditional rendering, some short circuit checking before it or the optional Chaining as it is shown in the suggestion

```suggestion
                        <p>Create by: <span>{beer.contributed_by?.split(" <")[0]}</span></p>
```

Reply:
yep just noticed after you added the empty beer earlier :)